tests/unit/s2n_client_hello_retry_test.c

```diff
@@ -168,8 +169,166 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_connection_free(server_conn));
             EXPECT_SUCCESS(s2n_connection_free(client_conn));
             EXPECT_SUCCESS(s2n_cert_chain_and_key_free(tls13_chain_and_key));
+        }
 
+#if !defined(S2N_NO_PQ)
```

Comment:
Do you need a version of this test that checks how this code behaves with S2N_NO_PQ defined?

Reply:
If `S2N_NO_PQ` is defined, no [KEMs, KEM groups](https://github.com/awslabs/s2n/blob/main/tls/s2n_kem.h#L79-L93), or [KEM preferences](https://github.com/awslabs/s2n/blob/main/tls/s2n_kem_preferences.h#L32-L48) are compiled in. So, there is nothing to test (except the classic ECDHE case, which should be covered by existing tests).